ingestion-beam/src/test/java/com/mozilla/telemetry/util/NormalizeTest.java

```diff
@@ -0,0 +1,47 @@
+package com.mozilla.telemetry.util;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class NormalizeTest {
+
+  @Test
+  /**
+   * See a selection of existing raw channel names and their normalized versions in BigQuery:
+   *
+   * <pre>{@code
+   * SELECT channel, normalized_channel, count(*)
+   * FROM `moz-fx-data-derived-datasets.telemetry.clients_daily_v6`
+   * WHERE true
+   *   and submission_date_s3 >= DATE('2018-07-01')
+   *   and normalized_channel != 'Other'
+   * group by 1, 2
+   * order by 3 desc
+   * LIMIT 1000
+   * }</pre>
+   */
+  public void channel() {
+    assertEquals("release", Normalize.channel("release"));
+    assertEquals("esr", Normalize.channel("esr"));
+    assertEquals("beta", Normalize.channel("beta"));
+    assertEquals("aurora", Normalize.channel("aurora"));
+    assertEquals("nightly", Normalize.channel("nightly"));
+
+    assertEquals("nightly", Normalize.channel("nightly-cck-ubuntu"));
+
+    assertEquals("Other", Normalize.channel("no"));
+    assertEquals("Other", Normalize.channel("esr52"));
+    assertEquals("Other", Normalize.channel("Release"));
```

Comment:
@trink @whd 

The existing lua implementation of channel name normalization doesn't force inputs to lower case. Do we want to preserve this behavior such that "Release" is normalized to "Other"?

Reply:
Absent a compelling reason one way or the other I would say keep existing behavior. It would be better to make this decision by data (e.g. if there are a sufficiently large number of pings that we would consider "release" with "Release" as their channel then we might consider making this change). This is the kind of thing that could happen at the "validation work week" next week but it's probably relatively unimportant as I would guess there are few pings with this property.

Reply:
```
SELECT count(channel)
FROM `moz-fx-data-derived-datasets.telemetry.clients_daily_v6` 
WHERE channel = 'Release'
AND submission_date_s3 = DATE("2019-04-15")
```

1676 users in one day. Not particularly significant. I'll keep the existing behavior.